sequence-diagram-model-generator/modules/sequence-diagram-model-generator-core/src/main/java/io/ballerina/sequencemodelgenerator/core/model/Constants.java

```diff
@@ -0,0 +1,58 @@
+/*
+ *  Copyright (c) 2023, WSO2 LLC. (http://www.wso2.com) All Rights Reserved.
+ *
+ *  WSO2 LLC. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+
+package io.ballerina.sequencemodelgenerator.core.model;
+
+/**
+ * Constants used in the sequence diagram model.
+ *
+ * @since 2201.8.0
```

Comment:
```suggestion
 * @since 2201.8.5
```

Please apply this in other applicable places as well